Nodejs/Product/TestAdapter/TestFrameworks/Angular/jasmineReporter.js

```diff
@@ -49,13 +58,9 @@ function isObject(value) {
         && !(value instanceof String)
 }
 
-var myReporter = {
-    specDone: result => {
-        // Communicate results through the console.
-        console.log(JSON.stringify(result, getCircularReplacer()));
-    }
-};
-
 jasmine.getEnv().addReporter(myReporter);
-jasmine.getEnv().it_ = itReplacement(jasmine.getEnv().it_);
-
+if (jasmine.getEnv().it_) {
```

Comment:
is it possible to get both attributes `it` and `it_`?

Reply:
The API for `it` and `it_` changed multiple times and depending on the version one or the other works. So we need to support both cases here.